octobot/constants.py

```diff
@@ -65,6 +66,15 @@
 COMMUNITY_TRADINGVIEW_WEBHOOK_BASE_URL = os.getenv(
     "COMMUNITY_TRADINGVIEW_WEBHOOK_BASE_URL", "https://webhook.octobot.cloud/tradingview"
 )
+COMMUNITY_EXTENSIONS_IDENTIFIER = "scaleway"
+COMMUNITY_EXTENSIONS_CHECK_ENDPOINT = os.getenv(
+    "COMMUNITY_EXTENSIONS_CHECK_ENDPOINT", "https://premium.octobot.cloud"
+)
+COMMUNITY_EXTENSIONS_CHECK_ENDPOINT_KEY = os.getenv(
+    "COMMUNITY_EXTENSIONS_CHECK_ENDPOINT_KEY", "TODO"
```

Comment:
Should we let a "TODO" here?

Reply:
yes, we need to add a key in the production function to remove this one